src/lib/Components/Bidding/Screens/ConfirmBid.tsx

```diff
@@ -343,14 +365,22 @@ export class ConfirmBid extends React.Component<ConfirmBidProps, ConfirmBidState
           </View>
 
           <View>
-            {requiresCheckbox && (
+            {requiresCheckbox ? (
```

Comment:
I know this is scope creep... but since I was finally able to test returning to this page I noticed that we weren't showing the "You agree to..." message when you were already registered. 😄

Reply:
👍